tests/jerry-test-suite/es2015/19/19.01/19.01.02/19.01.02-001.js

```diff
@@ -0,0 +1,41 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function test_set_prototype_of_error(o, proto, msg)
+{
+  var name = "";
+
+  try
+  {
+    Object.setPrototypeOf(o, proto);
+  }
+  catch (e)
+  {
+    name = e.name;
+  }
+
+  assert(name === "TypeError");
+
+  if (msg)
+  {
+    print(msg + " PASS (XFAIL)");
```

Comment:
I'd very much stick with these. They don't cause problems nor do they add complexity but give a very fast feedback if one of the tests fail.

(Note that these prints are swallowed by the testing infrastructure if the whole test passes so the output is not cluttered. But if some part of the test fails, it gives an immediate feedback on how far the test got, what parts succeeded and what was the first part to crash or assert out.)